py/Boinc/setup_project.py

```diff
@@ -339,6 +340,31 @@ def dir(*dirs):
             shutil.copy(srcdir('html/user/sample_motd.php'), dir('html/user/motd.php'))
         os.system("rm -f "+dir('html/languages/translations/*'))
         install_glob(srcdir('html/languages/translations/*.po'), dir('html/languages/translations/'))
+        try:
+            os.remove(srcdir('html/inc/release.inc'))
+        except OSError:
+            pass
+        try:
+            s = subprocess.Popen(["git", "rev-parse", "HEAD"], stdout=subprocess.PIPE, stderr=open(os.devnull, 'w'))
+            commit = s.stdout.read()[:7]
+            s = subprocess.Popen(["git", "branch"], stdout=subprocess.PIPE, stderr=open(os.devnull, 'w'))
```

Comment:
If you're in a detatched HEAD state (which will be the case for boinc-server-docker builds) this will give you something that looks like `(HEAD detached at 91ceaae7c8)` which is probably undesired. `git rev-parse --abbrev-ref HEAD` might be better, which will just return `HEAD` in that case instead. 

Actually though, what's the motivation for even storing the branch, given that we have the commit? I'd say we probably don't need the branch at all.

Reply:
I will remove the branch detection